src/index.html

```diff
@@ -5,8 +5,255 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#main-page" class="header__logo">
+        <img src="images/logo.png" alt="Logo Moyo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a href="#phones" class="nav__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">ГАДЖЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablet" class="nav__link">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">ФОТО</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="catalog-wrapper">
+        <div class="card" data-qa="card">
+          <img
+              class="card__product-image"
+              alt="imac-image"
+              src="./images/imac.jpeg"
+          >
+          <p class="card__product-name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+          <p class="card__product-code">Код товара: 195434</p>
+          <div class="card__rating-container">
+            <div class="card__product-stars stars-count-4">
```

Comment:
```suggestion
            <div class="card__product-stars stars stars--count-4">
```
You can't include modificator of a block without the original block. Modificators should be written with '--' or '_' symbols